agent/php_zval.h

```diff
@@ -343,13 +343,27 @@ static inline void nr_php_zval_bool(zval* zv, int b) {
  *           that function will then set the value).
  */
 static inline void nr_php_zval_prepare_out_arg(zval* zv) {
-#ifdef PHP7
+#if ZEND_MODULE_API_NO >= ZEND_7_0_X_API_NO /* PHP 7.0+ */
   ZVAL_NEW_REF(zv, &EG(uninitialized_zval));
 #else
   ZVAL_NULL(zv);
 #endif
 }
 
+/*
+ * Purpose : Wraps a zval to be ready for use as an out argument (ie an
+ *           argument that will be passed to a function by reference)
+ *           with a value.  Used for PHP8 since no longer automatically
+ *           wraps the argument.
+ */
+static inline void nr_php_zval_wrap_out_arg(zval* zv) {
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP 8.0+ */
+  if (!Z_ISREF_P(zv)) {
+    ZVAL_NEW_REF(zv, zv);
+  }
+#endif
+}
+
```

Comment:
This looks to be unused (and is causing build warnings on PHP < 8.0 which is leading to build failures). I'm going to remove it.